### PR TITLE
Print all available addresses at start up

### LINKED ISSUE #101: Log all addresses at startup
**Is your feature request related to a problem? Please describe.**
When debugging it is cumbersome to check what address an agent have.

**Describe the solution you'd like**
We can log all addresses used by the agent (skill & connection side) in a nice table at startup.

**Describe alternatives you've considered**
-

**Additional context**
-

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

This PR extends the run command to print all available addresses

## Fixes

#101 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/run.py [hunk 1]
```diff
@@ -196,10 +196,36 @@ def _profiling_context(period: int) -> Generator:
         sys.stderr = open(os.devnull, "w")
 
 
+def print_table(rows: Sequence) -> None:
```
Comment (angrybayblade) [inline aea/cli/run.py:199]: Extracted the function to print tables since we're printing a lot of tables

File: aea/cli/run.py [hunk 2]
```diff
@@ -214,73 +240,34 @@ def _print_instantiated_components(aea: AEA) -> None:
...
+
+
+def _print_addresses(aea: AEA) -> None:
+    """Print all the addresses used by agent."""
+
+    addresses = [
+        ("Name", "Address"),
+        *((k, v) for k, v in aea.context.addresses.items()),
+        ("decision_maker", aea.context.decision_maker_address),
+    ]
```
Comment (angrybayblade) [inline aea/cli/run.py:267]: @DavidMinarsch Are there any other addresses we should be printing?

**Review by angrybayblade — COMMENTED** [review]
```
All available addresses.
=================================================================
| Name           | Address                                      |
=================================================================
| ethereum       | 0x0603C88E423f4f9305491559dFa1c7e52cb0245B   |
| fetchai        | fetch1s5qz33dnhypjxyrv8rwmdh38d96c2w72l6eljz |
| decision_maker | decision_maker                               |
=================================================================
```

File: aea/cli/run.py (hunk 2 above)
Comment (DavidMinarsch) [reply aea/cli/run.py:267]: We don't need the decision_maker one. Otherwise this is good.

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
LGTM - please address comments and add tests.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**